console/src/main/scripts/plugins/metrics/ts/app-details/appServerJvmDetails.ts

```diff
@@ -126,36 +146,16 @@ module HawkularMetrics {
       });
     }
 
-    private autoRefreshPromise:ng.IPromise<number>;
-
-    private autoRefresh(intervalInSeconds:number):void {
-      this.autoRefreshPromise = this.$interval(() => {
-       this.refresh();
-      }, intervalInSeconds * 1000);
-
-      this.$scope.$on('$destroy', () => {
-        this.$interval.cancel(this.autoRefreshPromise);
-      });
-    }
-
-    public refresh():void {
-      this.getJvmData();
-      this.getAlerts(this.$routeParams.resourceId, this.startTimeStamp, this.endTimeStamp);
-    }
-
     public getJvmData():void {
-      this.endTimeStamp = this.$routeParams.endTime || +moment();
-      this.startTimeStamp = this.endTimeStamp - (this.$routeParams.timeOffset || 3600000);
-
       this.MetricsService.retrieveGaugeMetrics(this.$rootScope.currentPersona.id,
-        'MI~R~[' + this.$routeParams.resourceId + '~~]~MT~WildFly Memory Metrics~Heap Used',
+        'MI~R~[' + this.resourceId + '~~]~MT~WildFly Memory Metrics~Heap Used',
```

Comment:
In the constructor we have:

  this.resourceId = this.$routeParams.resourceId;

I just used the local var instead of going through routParams everywhere. Is that not OK?